test/onnx/test_fx_op_consistency.py

```diff
@@ -162,6 +162,54 @@
     ]
 )
 
+
+# NOTE: For ATen signature modifications that will break ONNX export,
+# use **xfail_onnxscript_forward_compatibility** and **skip_torchlib_forward_compatibility** instead of xfail or skip
+# to make the signal apparent for maintainers.
+def xfail_onnxscript_forward_compatibility(
+    op_name: str,
+    variant_name: str = "",
+    *,
+    reason: str,
+    github_issue: str,
+    opsets: Optional[Collection[Union[int, Callable[[int], bool]]]] = None,
+    dtypes: Optional[Collection[torch.dtype]] = None,
+    matcher: Optional[Callable[[Any], bool]] = None,
+    enabled_if: bool = True,
+):
```

Comment:
Maybe add a docstring explaining when to xfail? if we have an example to show as a scenario, that could help pytorch core to make a decision and unblock themselves

Reply:
Added docstring